staff/marta-penya/precourse/bingo/main.js

```diff
@@ -0,0 +1,193 @@
+let cartonBingo = [];   //variable para rellenar el cartón de bingo
+let cantidadNumeros = 15;  //números que vamos a introducir en el cartón 
+let numerosBombo = [];
+let rondas = 0;
+let stopprimera = 0;
+let stopsegunda = 0;
+let stoptercera = 0;
+
+
+//Cuando se ejecute, pedirá el nombre del jugador y deberá guardarse.
+
+function nombreUsuario(){
+  nombreU = prompt('¡Bienvenido/a al BINGO Skylab! ¿Cómo te llamas?')
+  nombreU ? alert(`Hola ${nombreU}!`) : alert('¡Hola Bingo player!');
+};
+
+
+
+//en el primer turno se mostrará un cartón con 15 números random (excluyendo el 0 siempre)
+
+function nuevoCarton(){
+
+  while(cartonBingo.length < cantidadNumeros){
+  let numeroAleatorio = Math.floor(Math.random() * 99) + 1;
+  let existe = false;
+  for(let i=0; i < cartonBingo.length; i++){
+	if(cartonBingo[i] === numeroAleatorio){
+      existe = true;
+      break;
+    }
+  }
+  if(!existe){
+   cartonBingo[cartonBingo.length] = numeroAleatorio;
+  }
+}
+}
+
+
+
+//mostrar el carton generado
+function mostrarCarton(){
+  primeraLinea = cartonBingo.slice(0,5)
+  segundaLinea = cartonBingo.slice(5,10)
+  terceraLinea = cartonBingo.slice(10,15)
+
+console.log("Tu cartón es:" + "\n" + primeraLinea + "," + "\n" + segundaLinea + "," + "\n" + terceraLinea)  
+}
+
+
+
+// para pasar al siguiente turno el usuario deberá confirmar mediante confirm() visualizandose otro número, si coincide 
+//con alguno de los existentes en el cartón, cambiará por una "X" o un 0
+
+
+function nuevoTurno(){
+  turno = confirm('¿Deseas un número?') 
+  turno ? numRandom() : alert('¡Hasta otra!')
+}
+
+
+
+//tachar numero que coincide con el carton 
+
```

Comment:
Would be super nice if code shows the how the bill is evolving with matched numbers :)

Reply:
Yeah, this bingo shows the bingo card but with console.log(), you can check it in the in the console of your browser